generic_tracers/generic_COBALT.F90

```diff
@@ -7873,11 +7871,12 @@ subroutine generic_COBALT_update_from_source(tracer_list,Temp,Salt,tv,rho_dzt,dz
     real, dimension(:,ilb:,jlb:,:), intent(in) :: opacity_band
     real, dimension(ilb:,jlb:),     intent(in) :: internal_heat
     real, dimension(ilb:,jlb:),     intent(in) :: frunoff
+    real, dimension(ilb:,jlb:),     intent(in) :: geolat
+    type(EOS_type),                 intent(in)  :: eqn_of_state !< Equation of state structure
```

Comment:
should these new variables be optional here like they are in generic_tracer_source ?